### PR TITLE
Italian: Fix format and added "throttle" support

### PR DESCRIPTION
**Pull request by billmn**

### CODE REVIEW

File: it/pagination.php [hunk 1]
```diff
@@ -14,7 +14,6 @@
     */
 
     'previous' => '&laquo; Precedente',
-
     'next'     => 'Successivo &raquo;',
 
-];
+];
```
Comment (GrahamCampbell) [inline it/pagination.php]: cs

File: it/passwords.php [hunk 2]
```diff
@@ -13,14 +13,11 @@
...
-
-];
+    'password' => 'Le password devono essere di almeno 6 caratteri e devono coincidere.',
+    'reset'    => 'La password è stata reimpostata!',
+    'sent'     => 'Promemoria della password inviato!',
+    'throttle' => 'Troppi tentativi di accesso. Riprova tra :seconds secondi.',
+    'token'    => 'Questo token per la reimpostazione della password non è valido.',
+    'user'     => 'Non esiste un utente associato a questo indirizzo e-mail.',
+
+];
```
Comment (GrahamCampbell) [inline it/passwords.php]: cs

File: it/validation.php [hunk 3]
```diff
@@ -104,4 +105,4 @@
 
     'attributes' => [],
 
-];
+];
```
Comment (GrahamCampbell) [inline it/validation.php]: cs

**Comment by billmn** [comment]
Fixed

File: it/validation.php [hunk 4]
```diff
@@ -34,8 +35,8 @@
     "digits"           => ":attribute deve essere di :digits cifre.",
     "digits_between"   => ":attribute deve essere tra :min e :max cifre.",
     "email"            => ":attribute non è valido.",
-    "exists"           => ":attribute selezionato/a non è valido.",
     "filled"           => ":attribute è richiesto.",
+    "exists"           => ":attribute selezionato/a non è valido.",
```
Comment (caouecs) [inline it/validation.php]: why ?

**Comment by caouecs** [comment]
I accept but I fix the error of order in validation

File: it/validation.php (hunk 4 above)
Comment (billmn) [inline it/validation.php]: Because Laravel have a different order. 

https://github.com/laravel/laravel/blob/master/resources/lang/en/validation.php

Comment (caouecs) [inline it/validation.php]: I think it's a problem on Laravel version

**Comment by billmn** [comment]
In any case we have to change labels again 

https://github.com/laravel/laravel/commit/ef958716c37fcdf0ea80bb89ba0c93394ae0b157

**Comment by billmn** [comment]
Oh I don't see that you have already updated the package, sorry

**Pull request merged**